File: week-04/day-2/src/garden/Tree.java
package garden;

public class Tree extends Plant {

    int numberOFTress;

    public Tree(int numberOFFlowers, String color, int waterNeed, double waterAbsorbtion){
        super(color , waterNeed, waterAbsorbtion);
        this.numberOFTress = numberOFFlowers;
        this.waterAbsorbtion = 0.40;
    }

}

File: week-10/day-4/Garden/Garden/Tree.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Garden
{
    class Tree : Plant
    {

        public Tree(String color, double waterNeed, double waterAbsorbtion) : base(color, waterNeed, waterAbsorbtion)
        {
        }
        public override void Status()
        {
            if (waterNeed < 10)
            {
                Console.WriteLine("The " + color + " " + this.GetType().Name + " needs water.");
            }
            else
            {
                Console.WriteLine("The " + color + " " + this.GetType().Name + " doesn't needs water.");
            }
        }
    }
}
